pkgs/top-level/all-packages.nix

```diff
@@ -26722,13 +26727,17 @@ with pkgs;
   };
   systemdStage1 = systemdMinimal.override {
     pname = "systemd-stage-1";
+    withAcl = true;
     withCryptsetup = true;
     withFido2 = true;
+    withKmod = true;
+    withPam = true;
     withTpm2Tss = true;
   };
   systemdStage1Network = systemdStage1.override {
     pname = "systemd-stage-1-network";
     withNetworkd = true;
+    withLibidn2 = true;
```

Comment:
`libidn2` only makes sense if the networking functions are enabled (resolved or networkd), so I have put it into the `systemdStage1Network` instead of the default `systemdStage1`. Is that reasonable?

Reply:
I think so, yes.